server/model/display-type.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+export type DisplayType = 
+  | "lead"
+  | "regular";
```

Comment:
Could we potentially be using [`weight`](https://github.com/wellcometrust/wellcomecollection.org/blob/ebbe1f5411d12f92f1d87f09e47de1bbe419dfb7/server/model/body-part.js#L3) here?

So far we have `supporting`, `standalone`, `default` (they're not `enum`ed yet) - but it could be good to start getting a unified language, thus logic, going.